cli/src/pcluster/validators/directory_service_validators.py

```diff
@@ -48,6 +48,25 @@ def _validate(self, domain_addr, additional_sssd_configs):
             self._add_failure(warning_message, FailureLevel.WARNING)
 
 
+class DomainNameValidator(Validator):
+    """Domain name validator."""
+
+    FQDN_PATTERN = "^([a-zA-Z0-9_-]+)(\\.[a-zA-Z0-9_-]+)*$"
```

Comment:
This accept ip addresses, is it intended to do so?

Reply:
IP addresses should not be accepted.
Will iterate on the above regexp

Reply:
The regexp you suggested regarding FQDN does not fit our case because:
1. it is a full LDAP DN, whereas we need here only the domain part
2. it accepts strings like `DC=corp,DC=mgiac/mo,DC=com`, which cannot be accepted

Reply:
I will take the FQDN regexp you suggested as it fits our case